diff --git a/internal/cli/root.go b/internal/cli/root.go
index de82940..0efbec7 100644
--- a/internal/cli/root.go
+++ b/internal/cli/root.go
@@ -1,6 +1,7 @@
 package cli
 
 import (
+	"github.com/carapace-sh/carapace"
 	"context"
 	"fmt"
 	"os"
@@ -115,6 +116,7 @@ func NewRootCmd(ctx context.Context, cliCtx *CliContext) *cobra.Command {
 	interactiveConnFlag.bind(rootCmd)
 
 	rootCmd.MarkFlagsMutuallyExclusive("no-password", "password")
+	carapace.Gen(rootCmd)
 
 	return rootCmd
 }
